src/geoarray.jl

```diff
@@ -2,7 +2,10 @@ struct GeoArray{T, N} <: AbstractArray{Union{Missing, T}, N}
     A::Array{Union{Missing, T}, N}
     f::AffineMap
     crs::AbstractString
+    GeoArray(A,f,crs) = length(size(A)) != 3 ? error("Array should be three-dimensional.") : new{T, N}(A,f,crs)
```

Comment:
If we force this type to be 3 dimensional, there is no need for the type parameter N right?
Or we do allow N dimensional `GeoArray` (btw is `GeoArray` or `GeoRaster` better, since the package is called `GeoRasters`), but certain methods related to GDAL IO are only supported for 2 or 3 dimensional data, and for those we assume GDAL style (col, row, band) ordering?

Reply:
I'm unsure about both. 😄 
`GeoArray` seems nice as a `AbstractArray`. I also think that having *raster* in the name helps understand the package.
I'm thinking of making it always 3 dimensional, with a method constructor for reshaping two dimensional datasets automagically.